requirements_test.txt

```diff
@@ -11,6 +11,7 @@ astroid==3.3.11
 coverage==7.10.6
 freezegun==1.5.2
 go2rtc-client==0.2.1
+jsonpickle==4.1.1
```

Comment:
Why do we depend on thios now?

Reply:
Oh I see now, we just have test helpers for that

Reply:
The rationale is:

I built a tool that can export the OpenRGBClient and OpenRGBDevice into JSON in a way that they can be restored back into OpenRGBClient and OpenRGBDevice classes.

This provides an easy way of dumping the OpenRGB data to build more accurate test cases.

My objects didn't use to source from JSON, but I have built this solution that allows it to, with the above-described benefit.

And that requires this new dependency.

Deleting the helper is a no-brainer, it's not being used by the tests anyway.

All that said, I want to clarify: are you asking me to revert these changes?

Reply:
I rewrote the logic in ee980a39d85f176f959f653107232ea8b3e76dc8 to avoid depending on jsonpickle. It's much simpler now.